docs/manual/install/index.rst

```diff
@@ -92,3 +92,36 @@ Stable versions of Qtile can be installed from PyPI:
 As long as the necessary libraries are in place, this can be done at any point,
 however, it is recommended that you first install xcffib to ensure the
 cairo-xcb bindings are built (see above).
+
+To recap, let's take it as an example on Fedora distro to run qtile within a virtualenv,
+the virtualenv is optional (assume that the git repo is cloned at ``~/local/qtile/`` of user ``foo``):
+
+#. (optional) Create a virtualenv ``~/local/qtile/qtile-env/``, and install the dependencies there.
+#. (optional) Create a glue shell to take advantage of the virtualenv.
```

Comment:
Is this the `/home/foo/local/qtile/qtile-venv-entry` file? Is that optional? May be good to note where the output should go.